mri_team_count/create_mri_team_count_series_view.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
```

Comment:
This is the python code that runs the `mri_team_count_series.sql` query to create materialized view.